### DISCUSSION TITLE
Provide HTTP status for external probe tools

### DISCUSSION THREAD

**Discussion by themightychris** [spatie/laravel-health#87] [Ideas]
Many external health probe tools, like Kubernetes' HTTP probes, need to be able to simply check whether the HTTP response status is 2xx or 3xx to know if the system is healthy. The current response body for the JSON endpoint is too complex for many systems to read, an external tool would have to parse and loop through the json to make a healthy/unhealthy determination

Similar to the `--fail-command-on-failing-check` option for the CLI check, an optional query param for the JSON check to make it return a `503` status instead of a `200` status if any checks are failing would be very helpful

**Comment by freekmurze** [comment]
Feel free to send a PR to add this to the package. Make sure to include tests and update docs.

**Comment by francoisauclair911** [comment]
@themightychris are you working on this ? We can team up if you want
https://github.com/spatie/laravel-health/discussions/90
